src/views/Deposit/DepositTable.tsx

```diff
@@ -16,11 +16,18 @@ import { TokenIcon, useNativeSymbols } from "../../utils/icons";
 import { useUserAssetBalance } from "../../queries/userAssets";
 import { isMobile } from "react-device-detect";
 import { useDecimalCountForToken } from "../../queries/decimalsForToken";
+import { NATIVE_TOKEN } from "../../queries/allReserveTokens";
 
-const BalanceView: React.FC<{ tokenAddress: string }> = ({ tokenAddress }) => {
+const BalanceView: React.FC<{ tokenAddress: string; symbol: string }> = ({
```

Comment:
Why are you taking symbol as a secondary input to determine whether or not it's talking about the native token, when you could just take `string | NATIVE_TOKEN` as the `tokenAddress` input?